lib/gitlab/ci/templates/jobs/security/FortifyScanning.yml

```diff
@@ -0,0 +1,28 @@
+variables:
+  JAVA_SRC_VERSION: 11
+  PACKAGE_ENTRY_POINT: "./**/*.java"
+  KUBERNETES_HELPER_MEMORY_REQUEST: 4G
+  KUBERNETES_HELPER_MEMORY_LIMIT: 4G
+
+fortify_scanning:
+  stage: test
+  image: ${TAK_REGISTRY_HOST}/devsecops/fortify-tooling:1.2.0
+  allow_failure: true
+  script:
+    - set -x
+    - fortifyupdate && sourceanalyzer -Xmx4096M -b $CI_JOB_NAME -clean && sourceanalyzer -Xmx4096M -b $CI_JOB_NAME -project-root ./.fortify -logfile "./${CI_JOB_NAME}-analyze.log" -exclude "./src/test/**/*.*" -source $JAVA_SRC_VERSION $PACKAGE_ENTRY_POINT && sourceanalyzer -Xmx4096M -b $CI_JOB_NAME -scan -project-root ./.fortify -logfile "./${CI_JOB_NAME}.txt" -f "./${CI_JOB_NAME}-scan_results.fpr"
```

Comment:
Should move '-exclude ".src/test/**/*.*"' to something that is more configurable.